### ISSUE TITLE
Training on Multi-GPUs

### ISSUE THREAD

**Issue by surya501** [carpedm20/BEGAN-tensorflow#9]
I am pretty new to tensorflow although I have some experience with Keras. If I have two GPUs in my system, would it be possible to accelerate the training? If so, how? Apologies if this is a obvious question.

**Comment by carpedm20** [comment]
Yes I think so. But it's quite dirty to support multiple GPUs and I don't have a server with such setting so it's hard to implement it :( But there are some good references for it like:
- https://github.com/tensorflow/models/tree/master/inception
- https://github.com/openai/pixel-cnn

**Comment by surya501** [comment]
Not, a exact solution, but a temp workaround -- which might be obvious to a lot of folks, but still jotting it down here in case it helps anyone.
Instead of using multiple GPUs for one training, I went another route... I invoked one session per GPU like this.
CUDA_VISIBLE_DEVICES=0 python main.py --dataset=Shoes --use_gpu=True
CUDA_VISIBLE_DEVICES=1 python main.py --dataset=CelebA --use_gpu=True
